tests/unit/babylon/src/Cameras/babylon.pointerInput.tests.ts

```diff
@@ -0,0 +1,1863 @@
+/**
```

Comment:
Really good!
These tests will definitely be helpful

Reply:
yea, i should have written the tests first.
They would have made refactoring `ArcRotateCameraPointersInput` much easier.

Reply:
I think I say that to me once a week ;D